<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="RENLTP2N103" failures="3" tests="5" name="logintest" time="45.801" errors="0" timestamp="2021-11-22T19:04:57 IST">
  <testcase classname="com.inetBanking.testCases.TC_LoginDDT_002" name="loginDDT" time="12.339"/>
  <testcase classname="com.inetBanking.testCases.TC_LoginDDT_002" name="loginDDT" time="4.392">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:29)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.inetBanking.testCases.TC_LoginDDT_002" name="loginDDT" time="5.835">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:29)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.inetBanking.testCases.TC_LoginDDT_002" name="loginDDT" time="5.999">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:29)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.inetBanking.testCases.TC_LoginDDT_002" name="loginDDT" time="11.192"/>
</testsuite> <!-- logintest -->
